tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/conversion/MavenConversionTest.java

```diff
@@ -271,4 +293,10 @@ private void finishConversionDialog(){
 		}
 	}
 
+	private static String getJavaVersion() {
```

Comment:
same as in: AbstractMavenSWTBotTest.java

Reply:
Removed.